### PR TITLE
docs: Added configuration for Minio as paperclip storage backend.

### PR DESCRIPTION
**Pull request by koolhead17**

### CODE REVIEW

File: docs/how-to-use-paperclip-with-minio.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# How to use Paperclip with Minio Server [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Paperclip is intended as an easy file attachment library for ActiveRecord. In this recipe we will learn how to configure  Minio as Object Storage backend for Paperclip.
```
Comment (GarimaKapoor) [inline docs/how-to-use-paperclip-with-minio.md]: Please add link to Paperclip.  Also change PaperClip to Paperclip. 

Edit the line : In this recipe 'you' will learn not 'we' will learn.
Object Storage should be in lower case. The line should be "Minio as an object storage backend"

File: docs/how-to-use-paperclip-with-minio.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Minio can be used to upload contents like images, music and docs with Paperclip as an object storage backend.
+
```
Comment (GarimaKapoor) [inline docs/how-to-use-paperclip-with-minio.md]: This line is not required. Repetitive

File: docs/how-to-use-paperclip-with-minio.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+## 1. Prerequisites
+
+Public Minio server for developers to use as a sandbox is at https://play.minio.io:9000. In this setup bucket ``mytestbucket``, access_key_id ``Q3AM3UQ867SPQQA43P2F``, secret_access_key ``zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG`` and ``play.minio.io:9000`` as  s3_host_name are used.
+
```
Comment (GarimaKapoor) [inline docs/how-to-use-paperclip-with-minio.md]: https://play.minio.io:9000 is running. Play is a hosted Minio server for testing and development purpose

File: docs/how-to-use-paperclip-with-minio.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+## 2. Installation 
+ 
+Install Paperclip from [here](https://github.com/thoughtbot/paperclip)
+
+## 3. Paperclip Configuration
+
```
Comment (GarimaKapoor) [inline docs/how-to-use-paperclip-with-minio.md]: In this setup bucket ``mytestbucket``, access_key_id ``Q3AM3UQ867SPQQA43P2F``, secret_access_key ``zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG`` and ``play.minio.io:9000`` as  s3_host_name are used

File: docs/how-to-use-paperclip-with-minio.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# How to use Paperclip with Minio Server [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Paperclip is intended as an easy file attachment library for ActiveRecord. In this recipe we will learn how to configure  Minio as Object Storage backend for Paperclip.
+
+Minio can be used to upload contents like images, music and docs with Paperclip as an object storage backend.
+
+## 1. Prerequisites
+
```
Comment (GarimaKapoor) [inline docs/how-to-use-paperclip-with-minio.md]: Minio Server is installed and running. Please follow Minio Quickstart guide <link> to install. 

This recipe uses https://play.minio.io:9000. Play is a hosted Minio server for testing and development purpose only.

Comment (GarimaKapoor) [inline docs/how-to-use-paperclip-with-minio.md]: Play (demo version) uses  access_key_id ``Q3AM3UQ867SPQQA43P2F``, secret_access_key ``zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG``

File: docs/how-to-use-paperclip-with-minio.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+Public Minio server for developers to use as a sandbox is at https://play.minio.io:9000. In this setup bucket ``mytestbucket``, access_key_id ``Q3AM3UQ867SPQQA43P2F``, secret_access_key ``zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG`` and ``play.minio.io:9000`` as  s3_host_name are used.
+
+## 2. Installation 
+ 
+Install Paperclip from [here](https://github.com/thoughtbot/paperclip)
+
+## 3. Paperclip Configuration
```
Comment (GarimaKapoor) [inline docs/how-to-use-paperclip-with-minio.md]: 3. Paperclip storage configuration

**Comment by koolhead17** [comment]
This can be closed via https://github.com/minio/cookbook/pull/108

File: docs/how-to-use-paperclip-with-minio.md (hunk 5 above)
Comment (GarimaKapoor) [inline docs/how-to-use-paperclip-with-minio.md]: @koolhead17 please add the (demo Version) in the text. Its missing in the present cookbook.

**Comment by GarimaKapoor** [comment]
Please remove the bullet mark in the explore further section of the cookbook.

**Comment by koolhead17** [comment]
@GarimaKapoor please review https://github.com/minio/cookbook/pull/109

**Comment by ZennouRachid** [comment]
I'm using minio for storing attachments, any one have an idea about how much hardware capacity should i have for this depending how much users i have!
for example: for this 3 cases:
- 10 users
- 200 users
- 1M users
thank you!

**Comment by harshavardhana** [comment]
@ZennouRachid we can best answer this on https://slack.minio.io - please join us there.. Thanks

**Comment by harshavardhana** [comment]
// cc @NitishT can you review?

**Pull request merged**